### PR TITLE
autoload.lua Configurable extension types

### PR DESCRIPTION
**Pull request by smarek**
Regarding the discussion in https://github.com/occivink/mpv-image-viewer/issues/5
we concluded, that using autoload should result in loading all of the media files in the directory, not just those listed in current EXTENSIONS Set.

Specifically image files (png,jpg,jpeg,tiff,gif,webp,bmp,svg and more) are in scope just as audio files, given that autoload.lua is not limited to video files.

I'm not lua programmer, but I've managed to do something that works in my environments

If allowing all file types right away would be too much of a hassle for the project, i suggest changing default option images to false.

### CODE REVIEW

File: TOOLS/lua/autoload.lua [hunk 1]
```diff
@@ -22,11 +25,30 @@ function Set (t)
...
+    for k in pairs(b) do res[k] = true end
+    return res
+end
+
+EXTENSIONS_VIDEO = Set {
+    'mkv', 'avi', 'mp4', 'ogv', 'webm', 'rmvb', 'flv', 'wmv', 'mpeg', 'mpg', 'm4v', '3gp'
+}
+
+EXTENSIONS_AUDIO = Set {
+    'mp3', 'wav', 'ogm', 'flac', 'm4a', 'wma', 'ogg', 'opus', 'jpg', 'png', 'tif', 'jpeg'
```
Comment (occivink) [inline TOOLS/lua/autoload.lua]: Stray image formats

Comment (smarek) [reply TOOLS/lua/autoload.lua]: damn, missed that, sorry

Comment (smarek) [reply TOOLS/lua/autoload.lua]: Fixed in https://github.com/mpv-player/mpv/commit/102d4a609effeb0433474d21ebf0fe048d228a0b

**Comment by smarek** [comment]
> I don't care about the contents of this script, but the commit prefix is wrong. I didn't wrote contribute.md for fun.

Sorry about that, didn't see the CONTRIBUTE.md in the top-level of repository (where it usually is placed), should be fixed with latest changes, i've provided documentation on top of the script, changed commit message to be informative and contain correct prefix.

**Comment by smarek** [comment]
Fixed, might be great to have those in CI tests, so one is directly lead to the cause, not bothering maintainers with such trivialities.

**Comment by wm4** [comment]
Not very easily checkable because there are exceptions, but PRs to do so would be accepted I guess.

**Comment by wm4** [comment]
>Sorry about that, didn't see the CONTRIBUTE.md in the top-level of repository (where it usually is placed),

Also, this is directly linked in the pull request template. You had to remove that link manually, or it would be in the first post.

**Pull request merged**